sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -143,6 +148,24 @@ above.
 
 # Use SYCL toolchain
 
+## Using the SYCL toolchain on CUDA platforms
+
+The SYCL toolchain support on CUDA platforms is still on an experimental phase.
+Currently, the SYCL toolchain relies on having a recent OpenCL implementation
+on the system in order to link applications to the SYCL runtime.
+The OpenCL implementation is not used at runtime if only the CUDA backend is
+active for the application.
+
+The OpenCL implementation provided by the CUDA SDK is OpenCL 1.2, which is
+too old to link with the SYCL runtime and lacks some symbols.
+
+We recommend to either install the low level CPU runtime,
```

Comment:
Do you mean:
We recommend either installing the low level CPU runtime, or following the instructions in the next section.

Reply:
No, the `either` doesn't make sense. Reworded.

Reply:
Addressed in 7d027efeda7845a8d6df6f7fa109d4c73451ea61